.github/workflows/ci.yml

```diff
@@ -1,29 +0,0 @@
-name: Node.js CI
-
-on: [push, pull_request]
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [14, 16, 18]
-
-    defaults:
-      run:
-        working-directory: ./web
-
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          submodules: recursive
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v2
-        with:
-          node-version: ${{ matrix.node-version }}
-      - run: yarn install
-      - name: Build FE for production test
-        working-directory: ./web/frontend
-        run: yarn install && yarn build
```

Comment:
I wonder if we want to delete this file - are there no checks worth doing? Even just building it?

Reply:
I've restored the file and just removed the test line